src/tools/gcc.lua

```diff
@@ -284,12 +284,20 @@
 -- Decorate include file search paths for the GCC command line.
 --
 
-	function gcc.getincludedirs(cfg, dirs, sysdirs)
+	function gcc.getincludedirs(cfg, dirs, sysdirs, frameworkdirs)
 		local result = {}
-		for _, dir in ipairs(dirs) do
+		for _, dir in ipairs(dirs or {}) do
```

Comment:
Is this change necessary? Is the `or {}` required on any of the blocks in this function?

Reply:
No, ti was (probably) made due to an invalid call of this function in a test. Test will be fixed